website/src/hooks/useApiCall.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2023 The Vitess Authors.
```

Comment:
Super nit - you could change this to 2024 😄 
```suggestion
Copyright 2024 The Vitess Authors.
```